src/Tile.jsx

```diff
@@ -27,7 +28,8 @@ const Tile = ({
     )}
     disabled={
       (minDate && minDateTransform(minDate) > date) ||
-      (maxDate && maxDateTransform(maxDate) < date)
+      (maxDate && maxDateTransform(maxDate) < date) ||
+      tileDisabled({ date, view })
```

Comment:
You should handle it like so:

```js
(tileDisabled && tileDisabled({ date, view }))
```

Reply:
:thumbsup: Yup, forgot that sanity check